databases/week2/1hyf_lesson2_class_exercise.sql

```diff
@@ -0,0 +1,38 @@
+-- Get all the tasks assigned to Pavel;
+SELECT
+  task.title,
+  user.name
+FROM
+  (
+    task
+    join user_task ON task.id = user_task.task_id
```

Comment:
if you like to use all keyword in capital, which I will suggest to do so, you may want to capitalize the JOIN too